ext/php8/signals.c

```diff
@@ -1,6 +1,5 @@
 #include "signals.h"
 
-#include "config.h"
```

Comment:
Doesn't `HAVE_SIGACTION` come from `config.h`?

Reply:
ah, I need to guard it with HAVE_CONFIG_H, you are right.